pyproject.toml

```diff
@@ -257,6 +245,18 @@ module = [
     "sentry.incidents.logic",
     "sentry.incidents.subscription_processor",
     "sentry.incidents.tasks",
+    "sentry.integrations.api.bases.external_actor",
+    "sentry.integrations.api.bases.integration",
+    "sentry.integrations.api.endpoints.group_integration_details",
+    "sentry.integrations.api.endpoints.group_integrations",
+    "sentry.integrations.api.endpoints.integrations.install_request",
+    "sentry.integrations.api.endpoints.organization.organization_code_mapping_codeowners",
+    "sentry.integrations.api.endpoints.organization.organization_code_mappings",
+    "sentry.integrations.api.serializers.doc_integration",
+    "sentry.integrations.api.serializers.external_actor",
+    "sentry.integrations.api.serializers.integration",
+    "sentry.integrations.api.serializers.integration_feature",
+    "sentry.integrations.api.serializers.rest_framework.doc_integration",
```

Comment:
as part of moving this can you fix the type errors?  this list isn't meant to be added to

Reply:
I believe all these files were originally in the mypy ignore, it's just that because they were moved the routes needed to be updated.

Reply:
I think one part of improving the organization would be to fix the typing while moving the files

Reply:
Could we fix the typing after the file moves? I think improving the typing as we're shifting the files around.